app/forms/support/recruitment_cycle_form.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Support
+  class RecruitmentCycleForm < ApplicationForm
+    include ActiveModel::Attributes
+
+    attribute :year, :string
+    attribute :application_start_date, :multiple_parameters_date
+    attribute :application_end_date, :multiple_parameters_date
+
+    validates :year, presence: true, numericality: { only_integer: true }
+    validates :application_start_date, :application_end_date, multiple_parameters_date: true
+    validate :application_end_date_must_be_after_start_date
+    validate :year_must_be_unique
+
+    def initialize(params = {})
+      processed_params = MultipleParametersDateType.process(params)
+
+      super(processed_params)
+    end
```

Comment:
As we spoke in our huddle. Something to look into removing in the future. No action for now.